src/scripts/components/main/globe-navigation/globe-navigation.tsx

```diff
@@ -45,16 +45,19 @@ const GlobeNavigation: FunctionComponent<Props> = ({
     <div className={styles.globeNavigation}>
       <Button
         className={styles.projection}
+        id="projection"
```

Comment:
As the ids must be globally unique we should prefix them to avoid potential future problems. Something like `btn-projection` or `ui-projection`